### ISSUE TITLE
voice like Mikey Mouse and fast

### ISSUE THREAD

**Issue by MrJamsn** [nirnachmani/ESP32-S3-Box-3-Firmware#5]
First of all, thank you for your awesome work!! 

I installed it correctly but have two issues, one only since I've installed this firmware.

The response voice is fast and high pitch, like mikey mouse. Is there any setting to speed up the reponses?

(second issue, maybe it is familiar. after a couple of minutes, the wake word doe not get recodgnized anymore. only after a restart it is operational again)

thank you!

**Comment by nirnachmani** [comment]
For the distorted voice, try to change the sample_rate of announcement_audio in the media_player block. The value in the original code was 16000 but this gave distorted voice (kind of opposite of what you are describing). Changing it to 22050 solved the problem. Maybe for you 16000 will work instead. 

I am not sure about the second problem. It doesn't happen with my unit. Try to look at the log as you say the wake word to see if something happens.

**Comment by MrJamsn** [comment]
Thank you, the sampling rate was the right place. 16000 works perfectly! Second problem, didn’t occur yesterday. Will check the logs.You know by any chance how to let it answer with a beep or tone if a entity was set? (Instead of “kitchenlight was turned on”)Thank you! ____________Raffael Jambor, ***@***.*** meinem iPhone gesendetAm 13.06.2024 um 11:55 schrieb nirnachmani ***@***.***>:﻿
For the distorted voice, try to change the sample_rate of announcement_audio in the media_player block. The value in the original code was 16000 but this gave distorted voice (kind of opposite of what you are describing). Changing it to 22050 solved the problem. Maybe for you 16000 will work instead.
I am not sure about the second problem. It doesn't happen with my unit. Try to look at the log as you say the wake word to see if something happens.

**Comment by nirnachmani** [comment]
Great. 

Regarding using a confirmation sound instead of a verbal answer, this kind of functionality is not part of the Box3 firmware. Instead, it is part of the assist pipeline that the boxes uses. It depends on what components you use in your assist pipeline. You may be able to implement a confirmation sound after you say the wake word (see [https://github.com/esphome/feature-requests/issues/2490](url)) but this is not what you are asking for.

**Comment by MrJamsn** [comment]
Thank you for the link! Will have a look :)

BR
